src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,62 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                id:
+                    description: Annotation ID
+                    type: string
+                private:
+                    description: If the annotation should only be visible to the author
+                    type: boolean
+                    default: false
+                hasParent:
+                    description: If the annotation has a parent annotation
+                    type: boolean
+                    default: false
+                title:
+                    description: Title of the annotated document
+                    type: string
+                target:
+                    description: Target text for the annotation
+                    type: string
+                targetUri:
+                    description: URI of the annotated document
```

Comment:
https://github.com/elifesciences/api-raml/pull/188#discussion_r146541874 is still relevant here.